cypress/e2e/pages/testsForAllCanonicalPages.js

```diff
@@ -2,13 +2,9 @@
 /* eslint-disable cypress/unsafe-to-chain-command */
 /* eslint-disable import/prefer-default-export */
```

Comment:
We can now remove this lint override because there is a default export

Reply:
Can we also check if the lint override on lines 1 & 2 are required and if not, remove them?